tests/ast/nodes/test_get_children.py

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
Is it important to test this API? Won't it be apparent that this doesn't work if we get other failures?

Reply:
Testing this behavior simplifies debugging of functionality that's reliant on it.  If a folding test fails but these tests don't, it's an area that we know not to look first.